gapis/api/vulkan/api/query_pool.api

```diff
@@ -41,10 +41,11 @@
 //////////////////////////
 
 enum QueryStatus {
-  QUERY_STATUS_INACTIVE      = 0
-  QUERY_STATUS_ACTIVE        = 1
-  QUERY_STATUS_COMPLETE      = 2
-  QUERY_STATUS_UNINITIALIZED = 4
+  QUERY_STATUS_CREATED = 0
+  QUERY_STATUS_RESET = 1
+  QUERY_STATUS_BEGAN = 2
+  QUERY_STATUS_ENDED = 3
+  QUERY_STATUS_TIMESTAMP_WRITTEN = 4
```

Comment:
I think that re-arranging this enumeration will break compatibility with old traces, since the old state values will be interpreted incorrectly.

If we keep the previous enumeration, then the state from old captures (prior to adding UNINITIALIZED) will mostly be correct except:
- QUERY_STATUS_INACTIVE will become QUERY_STATUS_RESET, even if it hadn't actually been initialized, which may cause an extra reset. A quick look at the spec makes me think that extra, unnecessary resets are safe.
- For captures made after the change ta vkCmdWriteTimestamp, and before this change, timestamp queries may have been in state QUERY_STATUS_COMPLETE (now QUERY_STATUS_ENDED) instead of QUERY_STATUS_TIMESTAMP_WRITTEN.